src/scripts.js

```diff
@@ -73,42 +84,46 @@ const showUserInfo = () => {
 };
 
 const showActivityInfo = () => {
-  activityInfo = new Activity(activityData);
-  today = activityInfo.returnLatestDate();
   const currentUserActivity = activityInfo.filterDataByUser(currentUser.id);
   const currentUserCurrentActivity = activityInfo.findDataByDate(today, currentUserActivity)
   const averageUserSteps = activityInfo.findAllUsersAverageByDate(today, 'numSteps');
   const averageUserMinutes = activityInfo.findAllUsersAverageByDate(today, 'minutesActive');
-  // const userWeeklySteps = activityInfo.returnUserWeeklyData(today, currentUser, 'numSteps');
+  const userWeeklySteps = activityInfo.findWeeklyActivityCounts(currentUser.id, today, 'numSteps');
+  const userWeeklyStairs = activityInfo.findWeeklyActivityCounts(currentUser.id, today, 'flightsOfStairs');
+  const userWeeklyMinutes = activityInfo.findWeeklyActivityCounts(currentUser.id, today, 'minutesActive');
   activityCurrentSteps.innerText = `${currentUserCurrentActivity.numSteps} steps`;
   activityCurrentMinutes.innerText = `${currentUserCurrentActivity.minutesActive} minutes`;
   activityCurrentMiles.innerText = `${activityInfo.calculateMiles(currentUser.id, today, userRepo)} miles`;
   activityAverageSteps.innerText = `${averageUserSteps} steps`
   activityAverageMinutes.innerText = `${averageUserMinutes} minutes`
-  activityAverageMiles.innerText = `${averageUserSteps} steps`
-  // activityWeekSteps.innnerHTML = `
-  // <li class="activity__steps-day1">${userWeeklySteps[0]}</li>
-  // <li class="activity__steps-day2">${userWeeklySteps[1]}</li>
-  // <li class="activity__steps-day3">${userWeeklySteps[2]}</li>
-  // <li class="activity__steps-day4">${userWeeklySteps[3]}</li>
-  // <li class="activity__steps-day5">${userWeeklySteps[4]}</li>
-  // <li class="activity__steps-day6">${userWeeklySteps[5]}</li>
-  // <li class="activity__steps-day7">${userWeeklySteps[6]}</li>
-  // `
+  let activityWeekSteps = document.querySelector('.activity__week-steps');
+  let activityWeekStairs = document.querySelector('.activity__week-stairs');
+  let activityWeekMinutes = document.querySelector('.activity__week-minutes');
+  let activityWeekDates = document.querySelector('.activity__week-dates');
+  activityWeekSteps.innerText = "Steps";
+  activityWeekStairs.innerText = "Flights of Stairs";
+  activityWeekMinutes.innerText = "Minutes";
+  activityWeekDates.innerText = "Dates";
+  activityWeekDates = printList(activityWeekDates, thisWeeksDates);
+  activityWeekSteps = printList(activityWeekSteps, userWeeklySteps);
+  activityWeekStairs = printList(activityWeekStairs, userWeeklyStairs);
+  activityWeekMinutes = printList(activityWeekMinutes, userWeeklyMinutes);
```

Comment:
Love the way we have this displaying with that dates to orient all those data points now - awesome job on this display, thanks for doing that!